react-components/src/architecture/concrete/pointsOfInterest/ads/PointsOfInterestAdsProvider.ts

```diff
@@ -165,7 +171,34 @@ function poiItemToInstance(item: PoiItem): PointsOfInterestInstance<ExternalId>
         externalId: item.sceneExternalId ?? 'dummy-scene-external-id',
         space: item.sceneSpace ?? 'dummy-scene-space'
       },
+      instanceRef: poiExternalInstanceRefToInstanceReference(item?.assetRef),
       sceneState: item.sceneState
     }
   };
 }
+
+function poiExternalInstanceRefToInstanceReference(
+  instance: PoiExternalInstanceRef | undefined
+): InstanceReference | undefined {
+  if (instance == undefined) {
+    return undefined;
+  }
+
+  if (isPoiAssetRef(instance)) {
+    return { assetId: instance.id };
```

Comment:
we always use the internal Id to manage assets in POis ? I saw some external ids flying around but I'm not fully sure 🤔 .

Reply:
Internal Ids are at least safer to use that external Ids, they're mandatory, but external IDs are not. Maybe the externallIds you saw is for DM instances? Also, the PoIs themselves use ExternalId